kind-east.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: cluster.example.dev/v1alpha1
```

Comment:
Not sure these new yaml files make sense in this PR. I assume they were in your repo for testing and were added with the rest of the changes ?

Reply:
Whoops, bitten by `git add .` again. Deleted.